data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -263,3 +257,36 @@ datasets:
         include_in_tiles: true
         include_in_downloadable_files: true
         create_percentile: true
+  - long_name: "NCLD Lack of Green Space / Nature-Deprived Communities dataset, as compiled by TPL"
```

Comment:
Nit: maybe spell out NCLD in the long-name?

Reply:
Done! ty

Reply:
(this is also a typo -- it's NLCD -- so ty for calling attention to it!)